packages/main/src/plugin/extension-loader.ts

```diff
@@ -465,19 +465,24 @@ export class ExtensionLoader {
       },
     };
 
-    const dialogs = this.dialogs;
     const progress = this.progress;
     const notifications = this.notifications;
     const inputQuickPickRegistry = this.inputQuickPickRegistry;
+    const messageBox = this.messageBox;
     const windowObj: typeof containerDesktopAPI.window = {
       showInformationMessage: (message: string, ...items: string[]) => {
-        return dialogs.showDialog('info', extManifest.name, message, items);
+        return messageBox.showMessage('info', extManifest.displayName, message, items);
       },
       showWarningMessage: (message: string, ...items: string[]) => {
-        return dialogs.showDialog('warning', extManifest.name, message, items);
+        return messageBox.showMessage('warning', extManifest.displayName, message, items);
       },
       showErrorMessage: (message: string, ...items: string[]) => {
-        return dialogs.showDialog('error', extManifest.name, message, items);
+        return messageBox.showMessage('error', extManifest.displayName, message, items);
+      },
```

Comment:
the API is accepting an optional MesageOptions items so we need to be handle that

Reply:
I was thinking of leaving these existing methods as-is to make it easy for extensions to create simple dialogs, or they can use the message box API for more complex cases.

Reply:
Out of date, replaced with new commit.